xbmc/cores/RetroPlayer/rendering/RPRenderManager.cpp

```diff
@@ -140,20 +150,16 @@ void CRPRenderManager::AddFrame(const uint8_t* data, size_t size, unsigned int w
     {
       std::vector<uint8_t> cachedFrame = std::move(m_cachedFrame);
 
-      if (!m_bHasCachedFrame)
```

Comment:
removing this check skips an important operation that elides a memory copy. is there a reason for this change? if unrelated to resolution changing it should go in a separate commit.

Reply:
As it is asynchronous, the cached frame could be updated? The memcopy will only happen when the emulator sends frames during pause.